scanner.go

```diff
@@ -359,3 +378,63 @@ func (s *scanner) closeRegionScanner() {
 	}
 	s.curRegionScannerID = noScannerID
 }
+
+// renews a scanner by resending scan request with renew = true
+func (s *scanner) renew() error {
+	if err := s.rpc.Context().Err(); err != nil {
+		return err
+	} else {
+		rpc, err := hrpc.NewScanRange(s.rpc.Context(),
+			s.rpc.Table(),
+			s.startRow,
+			nil,
+			hrpc.ScannerID(s.curRegionScannerID),
+			hrpc.NumberOfRows(s.rpc.NumberOfRows()),
```

Comment:
These shouldn't be needed on the RenewScan.

Reply:
Tested this out, we need the scannerId to identify the scanner we need to renew, but we don't need the number of rows so that will be removed.

Reply:
Oh, right. Of course the ScannerID is needed 🤦